cluster/src/main/scala/io/snappydata/impl/LeadImpl.scala

```diff
@@ -663,6 +669,15 @@ class LeadImpl extends ServerImpl with Lead
   }
 }
 
+class ExtendibleURLClassLoader(parent: ClassLoader)
```

Comment:
Theoretically, this classloader should replace install jar and its corresponding changes in job server and executors. We should track this and do the enhancement and do away with install jars. Else there will be too many options for jar loading.

Reply:
Also, this extension class does not do much. Can't it be a simple URLClassloader?

Reply:
Yes my intention is to remove install jar. That will be the next step. However in rowstore mode it should continue to work. That is why did not remove it eagerly.